defaults/main.yml

```diff
@@ -21,7 +21,12 @@ kubernetes_allow_pods_on_master: true
 kubernetes_enable_web_ui: true
 kubernetes_web_ui_manifest_file: https://raw.githubusercontent.com/kubernetes/dashboard/master/src/deploy/recommended/kubernetes-dashboard.yaml
 
-kubernetes_pod_network_cidr: '10.244.0.0/16'
+kubernetes_cni: 'calico'
+kubernetes_pod_network:
```

Comment:
I'm kind of torn on this 'hack' (what I'll call it) to loop through each network and only react on the one that matches based on a string condition.

I'd kind of rather have the user choose something like:

```yaml
kubernetes_pod_network:
  cni: 'calico'
  cidr: '10.244.0.0/16'
```

And then change the other vars accordingly. Maybe we also split out all CNI-specific stuff into its own task files? So one in `tasks/cni-flannel.yml`, another in `tasks/cni-calico.yml`? That way if someone also wants to add Weave support it's easy to see the pattern.